proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -63,6 +63,16 @@ message SignDoc {
   uint64 account_number = 4;
 }
 
+message SignDocAux {
+  bytes               body_bytes     = 1;
+  google.protobuf.Any public_key     = 2;
```

Comment:
Whose `public_key` is this in case of multiple signers? The last signer?

Reply:
This is the public key of the signer who is signing this sign doc.